app/helpers/vuelidate_form/vuelidate_form_helper.rb

```diff
@@ -30,24 +30,27 @@ def generate_form_using(method, args, options, block)
 	end
 
 	def set_options(options)
-		options[:builder] ||= VuelidateForm::VuelidateFormBuilder
+		builder = VuelidateForm::VuelidateFormBuilder
+		options[:builder] ||= builder
 		options[:html] ||= {}
 		options[:html].reverse_merge! ({
-			"@submit" => "vf.validateForm",
-			"@ajax:error" => "vf.handleError",
+			"slot-scope" => "#{builder::SLOT_SCOPE}",
+			"@submit" => "#{builder::SLOT_SCOPE}.validateForm",
+			"@ajax:error" => "#{builder::SLOT_SCOPE}.handleError",
 			"novalidate" => "",
 			"data-type" => "json"
 		})
 		options[:html][:autocomplete] = "off" unless options[:autocomplete]
 	end
 
 	def add_valid_form_wrapper(form_obj, form_text, form_opts)
-		validations = form_obj.validations.nil? ? "" : form_obj.validations.join(",")
-		toggles = form_obj.toggles.nil? ? "{}" : form_obj.toggles.to_json
+		validations = form_obj.validations.to_json
+		toggles = form_obj.toggles.to_json
 		content_tag("vuelidate-form", {
-			validate: validations,
-			:":start-toggles" => toggles,
-			:"v-slot" => "vf"
+			":validate" => form_obj.validations.to_json,
+			":start-toggles" => form_obj.toggles.to_json,
+			":value" => form_obj.value.to_json,
+			":error" => flash[:submission_error],
```

Comment:
why is `":validate"` here not `validations` that u just made right above?

Reply:
some idea I had about naming confusion because validations is the vue component option used to make vuelidate work, but I could probably also call this validations without any collision